spec/models/spotlight/resource_spec.rb

```diff
@@ -48,6 +48,30 @@ def to_solr
 
         subject.reindex
       end
+
+      it 'returns the number of indexed objects' do
+        allow(subject.send(:blacklight_solr)).to receive(:update)
+
+        expect(subject.reindex).to eq 1
+      end
+
+      it 'triggers a solr commit' do
```

Comment:
We're only committing after all indexing. Let me update the test to try to demonstrate that.